tests/kuttl/tests/keystone_scale/02-assert.yaml

```diff
@@ -0,0 +1,134 @@
+  containerImage: quay.io/tripleozedcentos9/openstack-keystone:current-tripleo
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  databaseUser: keystone
+  debug:
+    bootstrap: false
+    dbSync: false
+    service: false
+  passwordSelectors:
+    admin: AdminPassword
+    database: KeystoneDatabasePassword
+  preserveJobs: false
+  region: regionOne
+  replicas: 1
+  resources:
+    requests:
+      cpu: "1.0"
+      memory: 500Mi
+  secret: osp-secret
+status:
+  apiEndpoint:
+    admin: http://keystone-admin-openstack.apps-crc.testing
+    internal: http://keystone-internal-openstack.apps-crc.testing
+    public: http://keystone-public-openstack.apps-crc.testing
+  databaseHostname: openstack
+  readyCount: 1
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    openshift.io/scc: anyuid
+  labels:
+    service: keystone
+  #name: keystone-69f88774bd-gjspj   Can't include name due to random append
+  namespace: openstack
+spec:
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: service
+              operator: In
+              values:
+              - keystone
+          topologyKey: kubernetes.io/hostname
+        weight: 1
+  containers:
+  - args:
+    - -c
+    - /usr/local/bin/kolla_set_configs && /usr/local/bin/kolla_start
+    command:
+    - /bin/bash
+    image: quay.io/tripleozedcentos9/openstack-keystone:current-tripleo
+    imagePullPolicy: IfNotPresent
+    livenessProbe:
+      failureThreshold: 3
+      httpGet:
+        path: /v3
+        port: 5000
+        scheme: HTTP
+      initialDelaySeconds: 3
+      periodSeconds: 3
+      successThreshold: 1
+      timeoutSeconds: 5
+    name: keystone-api
+    readinessProbe:
+      failureThreshold: 3
+      httpGet:
+        path: /v3
+        port: 5000
+        scheme: HTTP
+      initialDelaySeconds: 5
+      periodSeconds: 5
+      successThreshold: 1
+      timeoutSeconds: 5
+    resources:
+      requests:
+        cpu: "1"
+        memory: 500Mi
+  initContainers:
+  - args:
+    - -c
+    - /usr/local/bin/container-scripts/init.sh
+    command:
+    - /bin/bash
+    env:
+    - name: DatabasePassword
+      valueFrom:
+        secretKeyRef:
+          key: KeystoneDatabasePassword
+          name: osp-secret
+    - name: AdminPassword
+      valueFrom:
+        secretKeyRef:
+          key: AdminPassword
+          name: osp-secret
+    - name: DatabaseHost
+      value: openstack
+    - name: DatabaseName
+      value: keystone
+    - name: DatabaseUser
+      value: keystone
+    image: quay.io/tripleozedcentos9/openstack-keystone:current-tripleo
+    imagePullPolicy: IfNotPresent
+    name: init
+    resources: {}
+  restartPolicy: Always
+  serviceAccount: keystone-operator-keystone
+  serviceAccountName: keystone-operator-keystone
+status:
+  phase: Running
```

Comment:
like above, almost all other service operators need keystone. and they'd call `make keystone_deploy` to get it up. I think we should have a corresponding `make keystone_deploy_validate` which uses this assert file to check. wondering if it is best to keep this in a common place/name and just link it here to 02-assert.yaml and then could use the common one in `make keystone_deploy_validate`?

Reply:
or as above copy it then to the consuming operator.